python/lsst/pipe/base/pipelineIR.py

```diff
@@ -416,20 +465,29 @@ class ImportIR:
     declared instrument prior to import.
     """
 
-    def toPipelineIR(self) -> "PipelineIR":
+    def toPipelineIR(self, directory: Optional[ButlerURI] = None) -> "PipelineIR":
         """Load in the Pipeline specified by this object, and turn it into a
         PipelineIR instance.
 
+        Parameters
+        ----------
+        directory : `ButlerURI` or `None`
```

Comment:
I think we should be careful if someone somehow creates a resourcePath that is somehow already defined as a relative URI to somewhere and paths end up getting confused.

Reply:
Can you give any example?  If URIs within the pipeline are already absolute, this won't do anything (just like `os.path.join("whatever", "/absolute/path") == "/absolute/path"`, and the behavior here shouldn't care whether `directory` here is relative or not; it'll just put a relative or absolute prefix on appropriately.